.github/workflows/docs-update.yml

```diff
@@ -0,0 +1,36 @@
+name: Update OpenTelemetry Website Docs
+
+on:
+  # triggers only on a manual dispatch
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: make-pr
+        env:
+          API_TOKEN_GITHUB: ${{secrets.GITHUB_TOKEN}}
```

Comment:
`GITHUB_TOKEN` is the repo's automated secret, not a PAT. I guess it won't be able to push to the destination repo. Is it possible to work with open-telemetry admins to have an org-scoped secret with a PAT for docs pushes that is exposed to our (and other) repo?

Reply:
hm, yeah, that's a good point. I assumed that since it was a public repo in the same org the permissions would be fine. Let me open an issue in the community repo to get a token.